.github/workflows/build_rpms.yaml

```diff
@@ -0,0 +1,33 @@
+name: Test building of HTCondor-CE RPMs
+on: [pull_request]
+
+jobs:
+  build-rpms-and-upload:
+    runs-on: ubuntu-latest
+    if: startsWith(github.repository, 'htcondor/')
+    strategy:
+      matrix:
+        dver: [7]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Prepare Docker
+        run: |
+          echo 'DOCKER_OPTS="-H tcp://127.0.0.1:2375 -H unix:///var/run/docker.sock -s devicemapper"' | sudo tee /etc/default/docker > /dev/null &&
+            sudo service docker restart
+
+      - name: Start CentOS ${{ matrix.dver}} image
+        run: |
+          docker run --detach --env "container=docker" \
+                 --name $GITHUB_SHA \
+                 --volume `pwd`:/htcondor-ce:rw \
+                 centos:centos${{ matrix.dver }} \
+                 /usr/sbin/init
+
+      - name: Build CHTC EL${{ matrix.dver }} RPMs
+        run: |
+          docker exec $GITHUB_SHA \
```

Comment:
Sorry where does this `$GITHUB_SHA` come from?

Reply:
It seems a bit confusing to me to use the git commit sha as the docker image name - like, "wait is that github sha a docker image hash, or just a name?  It's a name but it's not obvious when you look at it.
But i guess if that's the model you want then it works.

Even tacking on a prefix or suffix to the name could make it a bit more clear.  Eg, `auto-build.$GITHUB_SHA` - or some more descriptive name.